src/language-css/parser-postcss.js

```diff
@@ -438,21 +437,27 @@ function parseNestedCSS(node) {
   return node;
 }
 
-function parseWithParser(parser, text, frontMatter) {
+function parseWithParser(parser, text) {
   let result;
+  const frontMatterMatches = text.match(/^---\n([\s\S]*)---/);
```

Comment:
Maybe `/^---(\n[\s\S]*)?\n---/` ??

Reply:
Might work, will try it out :)

Reply:
fwiw, this will fail on this:

```
---
foo: bar
---

This is content.

---

and more content...
```

Since the regex will match up to the last `---`.

Try changing the regex to this: `/^---\n([\s\S]*?)---/` (add a `?`, to match lazily instead of greedily).

Reply:
Hey Jon, I tried those 2 cases and the output is the same. But indeed the current regex matched up to the second `---` so it's probably still better for it to match lazily. Making a small follow up PR. Nice catch 👍